t/ui/14-dashboard-parents.t

```diff
@@ -122,13 +122,23 @@ disable_bootstrap_animations();
 
 ok($driver->find_element('#group1_build13_1-0091 .h4 a')->is_displayed(), 'link to child group displayed');
 my @links = $driver->find_elements('.h4 a', 'css');
-is(scalar @links, 11, 'all links expanded in the first place');
+is(scalar @links, 16, 'all links expanded in the first place');
```

Comment:
this changes the test a little unmotivatedly. Why 5 more?

Reply:
Because the existing test is designed to just count the links on the page. Now, the the amount of links increased. It is the simplest change.

Reply:
Maybe you can find a more specific selector to get only the first link in each `.h4`. But if not that's not too bad in my opinion if you add a comment that this sum contains 5 of those 'hover links'.

Reply:
One magic number gets replaced by another one. But @Martchus suggestion still seems feasible.
In plain CSS I'd do it like this:
```
my @links = $driver->find_elements('.h4 a:first-child', 'css');
```

Reply:
Outdated, as I've changed the implementation. So, the test remains the same, with 11:)